QUARKUS-5658.md

```diff
@@ -0,0 +1,33 @@
+# QUARKUS-5658: Integrate OpenTelemetry to the WebSockets Next extension
+
+Jira: https://issues.redhat.com/browse/QUARKUS-5658
+
+This feature is specifically OpenTelemetry integration part of the new WebSockets Next extension.
+
+## Scope of the testing
+
+### Upstream testing
+* Unit tests have been added with implementation of this integration. Those 
+  unit tests verify that the correct spans are created on opening and closing
+  of the websocket connection.
+
+### General
+* An integration test will be added to Quarkus QE test suite's `monitoring`
```

Comment:
This is probably wrong, the `monitoring` is just parent, you will need to be more specific with concrete module you plan to use.